SurfaceTopography/Uniform/common.py

```diff
@@ -134,13 +134,12 @@ def _trim_nonperiodic(arr, scale_factor, op):
         r = -scale_factor * max(0, r - 1)
         if r == 0:
             r = None
-        trimmed_slice += [slice(L, r)]
+        trimmed_slice += [slice(int(L), int(r))]
```

Comment:
`int(None)` doesn't work:
```python
In [1]: int(None)
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-1-af2123a46eb2> in <module>
----> 1 int(None)

TypeError: int() argument must be a string, a bytes-like object or a number, not 'NoneType'
```
This can happen here?!

Reply:
Good catch! I think it can but there is no test for it... I just fixed the code